R/creds.R

```diff
@@ -59,6 +61,30 @@ fetch_creds <- function(dbname = "main-app", ...) {
   field_values
 }
 
+get_env_overwrite <- function(key, value) {
+  # Don't overwrite the "drv" field
+  if (key == "drv") {
+    return(value)
+  }
+
+  # Use fields list to create environment variable (e.g. DBCONNECT_ENDPOINT)
+  env_variable <- tryCatch(
+    paste0("DBCONNECT_", toupper(fields[[key]])),
+    error=function(...) NULL
+  )
```

Comment:
I am not clear why a `tryCatch` is needed here. Is it because the key might not exist in `fields`?

Reply:
Yes indeed